vsintegration/tests/FSharp.Editor.Tests/CodeFixes/CodeFixTestFramework.fs

```diff
@@ -208,16 +202,7 @@ module Xunit =
         let expectedLines = split expected
         let actualLines = split actual
 
-        if expectedLines.Length <> actualLines.Length then
-            Assert.Equal(expected, actual)
-        else
-            try
-                Assert.All(
-                    (expectedLines, actualLines) ||> Array.zip |> Array.rev,
-                    (fun (expected, actual) -> Assert.Equal(expected, actual))
-                )
-            with :? Xunit.Sdk.AllException as all when all.Failures.Count = 1 ->
```

Comment:
Yeah, `AllException` changed and does not expose `Failures` anymore in newer xunit.

Reply:
Wait so will this do equivalent this now? (probably not a big deal if not, just asking)

Reply:
It will print out the collections and point to the difference in the message, should do the work.